1.8/usage/cli/update.md

```diff
@@ -12,17 +12,56 @@ You can update the DC/OS CLI to the latest version or downgrade to an older vers
 
 You can upgrade an existing DC/OS CLI installation to the latest build.
 
-1.  Remove the current CLI binary:
-
+1.  Remove the current CLI binary. By default, this file is located in `/usr/local/bin/dcos`.
```

Comment:
we recommend instead of default? default is really 'they copied the UI instructions blindly`